### PR TITLE
Set a few arrays to zero

### PR DESCRIPTION
**Pull request by ndkeen**
Set a few arrays to zero. For some we found they were being used without init,others precautionary for DEBUG runs.

### CODE REVIEW

**Comment by bartgol** [comment]
Are these changes visible to the tests in `src/physics/p3/tests`? If not, then let's tell the AT to skip tests..

**Comment by ndkeen** [comment]
Sorry, first time to create PR. I don't know if this would affects tests.

**Comment by bartgol** [comment]
> Sorry, first time to create PR. I don't know if this would affects tests.

No worries Noel! This was more of a question for P3 developers. Anyhow, it seems that the AT wasn't really busy, and it already went through this PR.

**Comment by PeterCaldwell** [comment]
Thanks for the fix, Noel. These changes were on the F90 side, so weren't tested by AT anyways.

File: components/cam/src/physics/cam/micro_p3_interface.F90 [hunk 1]
```diff
@@ -1001,6 +1001,12 @@ subroutine micro_p3_tend(state, ptend, dtime, pbuf)
     !==============
     ! TODO: get proper value for 'it' from time module
     dummy_out(:,:) = 0.0_rtype
+    prt_liq = 0.0_rtype
+    prt_sol = 0.0_rtype
+    prec_pcw = 0.0_rtype
+    snow_pcw = 0.0_rtype
+    vap_liq_exchange = 0.0_rtype
```
Comment (PeterCaldwell) [inline components/cam/src/physics/cam/micro_p3_interface.F90:1008]: vap_liq_exchange was removed by #421 so I think this last line needs to be deleted/you should merge in master before merging to master...

**Comment by PeterCaldwell** [comment]
@ndkeen - for future reference, your BFB PRs won't get merged unless you add the AT:AUTOMERGE flag at the top right of this page. This tells the auto-tester (AT) to merge to master. If you have a non-bfb PR, you need to ask someone with merge permissions (Sandia folk or me) to merge it for you once you're confident is is working correctly.

File: components/cam/src/physics/cam/micro_p3_interface.F90 (hunk 1 above)
Comment (ndkeen) [reply components/cam/src/physics/cam/micro_p3_interface.F90:1008]: I think `vap_liq_exchange` is there. You may be thinking of `vap_cld_exchange`, which indeed was removed recently.

Comment (PeterCaldwell) [reply components/cam/src/physics/cam/micro_p3_interface.F90:1008]: You're totally right. I misread and misremembered my own PR. Sheesh.

**Pull request merged**